src/styles/index.css

```diff
@@ -0,0 +1,14 @@
+@import url('https://fonts.googleapis.com/css2?family=Abel&family=Signika:wght@600&display=swap');
+
+body {
+  margin: 0;
+  background-color: #eceef1;
+  font-family: 'Signika';
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family: 'Abel', sans-serif;
+  font-family: 'Signika', sans-serif;
+}
```

Comment:
Since you arent using the `code` selector anywhere in the project, this class could be removed